config/locales/en.yml

```diff
@@ -885,6 +886,45 @@ en:
       participate_in_campaign: Participate in a Campaign
       save_cloned_course: Save
       start_date: Start of activity tracking
+      scoping_methods:
+        scoping_methods: Scoping Methods
+        configure: Configure Scoping Methods
+        about: >
+          This is what lets the dashboard know which articles you wish to track. The articles tracked contribute to the course statistics and are displayed on the articles tab. Rather than adding each article you wish to track individually, you can add filters to track several of them at once.
```

Comment:
"For an Article Scoped Program, the Dashboard will only track edits to the articles that you have configured it for. After your page has been created, you can add individual articles to track assigning articles to individual editors or by adding them as Available Articles. For larger sets of articles, you can use any of these tools to define articles you want to track."